webdriver-spec.html

```diff
@@ -1314,6 +1320,12 @@
     return an <a>error</a> with code <a>timeout</a>.
   </ol>
 
+ <li><p>Set the <a>current browsing context</a> to the
+  <a>current top-level browsing context</a> as if <a>Switch to Frame</a>
+  command was executed with no specified
+  <a href="https://html.spec.whatwg.org/#child-browsing-context">
+  child browsing context</a>.
```

Comment:
I don’t think we should make a reference to **Switch To Frame** at all here.  If we need some generalised prose related to resetting the **current browsing context** we should add that as a definition, but I don’t think we do.